test/context-config.test.js

```diff
@@ -41,35 +41,41 @@ test('config', t => {
     handler
   })
 
-  fastify.inject({
-    method: 'GET',
-    url: '/get'
-  }, (err, response) => {
-    t.error(err)
-    t.equal(response.statusCode, 200)
-    t.same(JSON.parse(response.payload), Object.assign({ url: '/get', method: 'GET' }, schema.config))
-  })
+  {
```

Comment:
Why are we wrapping everything in blocks?

Reply:
I’m using `Block statements` to limit the scope of temporary variables like `response`. This avoids name conflicts and keeps the code cleaner.

Reply:
I think they are adding unnecessary indentation. These tests can be written like:

```js
let response = await aRequest()
assert.somethingAbout(response)

/* ... */

response = await aSecondRequest()
assert.somethingAbout(response)
```

Reply:
I usually use `block statement` if I want to use the same named variable instead of `let` assignment.
Both of them is acceptable, and using `block statement` is better in copy and paste to new test (not needed to rename `const` to `let`).

Reply:
Fair reasoning. It is clearly a matter of preference.

Reply:
I usually use blocks to reuse variables, and I followed this approach here since I noticed it’s used in other parts of the codebase.